README.md

```diff
@@ -109,6 +114,7 @@ to see all available versions.
 |------------------|-----------------------------|-----------------------------------------------|
 | Nicholas Strydom | nstrydom@gmail.com          | [nstrydom2](https://github.com/nstrydom2)     |
 | hentai-chan      | dev.hentai-chan@outlook.com | [hentai-chan](https://github.com/hentai-chan) |
+| Stefan Greve     | greve.stefan@outlook.jp     | [StefanGreve](https://github.com/stefangreve) |
```

Comment:
Honestly, this seems a bit distasteful. Maybe distasteful isn't the right word, however its a bit clunky, and can be slightly confusing from a third-party viewpoint. I would have preferred if contributors' work were submitted from one account.I guess it really doesn't matter all that much. This just looks weird to me. Do you plan on continuing contributions through the "Stefan Greve" profile?